test-configs.yaml

```diff
@@ -442,6 +442,11 @@ device_types:
     name: 'hi6220-hikey'
     mach: hisilicon
     class: arm64-dtb
+
+  hsdk:
+    name: 'hsdk'
+    mach: arc
+    class: arc-dtb
     boot_method: uboot
```

Comment:
This `boot_method: uboot` is for the entry above `hikey_uboot`, the `hsdk` entry was added in the middle of it.